### DISCUSSION TITLE
Renovate is not detecting java version upgrade

### DISCUSSION THREAD

**Discussion by deekshagpt** [renovatebot/renovate#19209] [Request Help]
Below is the entry in my pom.xml- 
<java.version>11</java.version>
but ronavate is not creating ny merge request for java version upgrade.
Could you please suggest what can be added for the same?

**Comment by viceice** [comment]
that's expected, as it's simply not implemented.

you can use the regex manager to update that value

**Reply by deekshagpt** [reply]
Could you please share the exact regex code snippet/example to update Java version?

**Comment by rarkins** [comment]
Seems like there are many ways to specify java version in `pom.xml`: https://www.baeldung.com/maven-java-version

**Reply by deekshagpt** [reply]
yes, I have used "java.version property"

**Reply by deekshagpt** [reply]
@rarkins what configurations would be required in renovate json file to check for java version updates?
